.github/workflows/OCV-PR-4.x-W10-Vulkan.yaml

```diff
@@ -0,0 +1,130 @@
+name: OCV PR:4.x W10 Vulkan
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/OCV-PR-4.x-W10-Vulkan.yaml'
+  workflow_call:
+
+concurrency:
+  group: OCV-PR-4.x-W10-Vulkan-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  EXTRA_CMAKE_OPTIONS: '-DCL_Z_OPTION=/Z7 -DOPENCV_DOWNLOAD_PATH=%BINARIES_CACHE% -DBUILD_EXAMPLES=ON -DOPENCV_ENABLE_NONFREE=ON -DCMAKE_BUILD_TYPE=Release -DWITH_OPENCL=OFF -DWITH_VULKAN=ON'
```

Comment:
We do not have Vulkan specific tests, so `-DBUILD_EXAMPLES=ON` is redundant.

Reply:
fixed.